recipients_app/lib/view/widgets/survey/survey_card_container.dart

```diff
@@ -1,20 +1,22 @@
-import "package:app/data/models/survey/survey_card_status.dart";
-import "package:app/data/models/survey/survey_ui_state.dart";
+import "package:app/core/cubits/survey/survey_cubit.dart";
+import "package:app/data/models/survey/mapped_survey.dart";
 import "package:app/ui/configs/configs.dart";
+import "package:app/view/widgets/dashboard_item.dart";
 import "package:app/view/widgets/survey/survey_card_bottom_action.dart";
 import "package:flutter/material.dart";
 
-class SurveyCard extends StatelessWidget {
-  const SurveyCard({super.key});
+class SurveyCardContainer extends DashboardItem {
+  final MappedSurvey mappedSurvey;
+  final SurveyCubit surveyCubit;
```

Comment:
I think you dont need to pass the surveyCubit here, because it is sitting on top of materialApp, isnt it? You have access to it from everywhere by context.read<SurveyCubit>(). If it's not on the top, I think I personally would put it on top or just in the "survey_card_bottom_action.dart" _navigateToSurvey method. But there is no need to pass it through all the widgets.